.gitignore

```diff
@@ -1,6 +1,5 @@
-.project
-.classpath
-.settings
-test-output
-target
-bin
+/bin/
+/target/
+/.settings/
+/.classpath
+/.project
```

Comment:
Sometimes the TAP integration, while playing with it, creates the test-output folder. Any reason to remove it here? We can include the /bin/ folder without removing it I think?